governance-and-automation/security-compliance-checks/tenant-storage.md

```diff
@@ -0,0 +1,35 @@
+---
+description: This article provides information on the Tenant Storage Limit report.
+---
+
+
+# Tenant Storage Limit
+
+Syskit Point mitigates the risk of running out of storage based on the settings defined in the tenant-wide [policy](../automated-workflows/tenant-storage-admin.md). 
+
+The purpose of the Tenant Storage Limit check is to **notify you when your storage percentage gets close to the limit you set**. This information can help you determine whether you should change the storage limit set for the policy, remove any redundant content to free up space or purchase additional storage from Microsoft.
+
+The Tenant Storage policy is **a tenant-wide policy, and it cannot be automated**. That means that no tasks are created to resolve this policy violation, and no emails are sent to collaborators.
+
+The tenant storage limits **are still detected** and shown on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails. 
```

Comment:
The tenant storage limits **are still detected** and shown mi čudno zvuči
Prijedlog:
When a tenant storage limit is exceeded, Syskit Point detects the policy vulnerability and shows it on the Security and Compliance dashboard.